model.py

```diff
@@ -1309,7 +1309,6 @@ class CoverageRecord(Base, BaseCoverageRecord):
 
     SET_EDITION_METADATA_OPERATION = u'set-edition-metadata'
     CHOOSE_COVER_OPERATION = u'choose-cover'
-    SYNC_OPERATION = u'sync'
```

Comment:
My circulation branch removes the CoverageProvider that provides this operation, but there's some other code (the reaper) that looked for the CoverageRecords it creates. Removing the constant ensures that I changed all of that code.